<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>761</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Угнанные автомобили:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Изменить данные в таблицах</string>
     </property>
     <addaction name="action_6"/>
     <addaction name="action_7"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Добавление новых данных о угоне</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Вывод наиболее угоняемых ТС</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Вывод самых опасных мест в городе</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Вывод используемых способов угона</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Таблица &quot;угнанное тс&quot;</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Таблица &quot;Данные о ТС&quot;</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Таблица &quot;способ угона&quot;</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Обновить данные в этом окне</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
